python/tests/test_session.py

```diff
@@ -29,6 +29,8 @@
 import graphscope
 from graphscope.client.session import DEFAULT_CONFIG_FILE
 
+graphscope.set_option(show_log=True)
```

Comment:
Put the global setting to `setUpModule`.

c.f.: https://docs.python.org/3/library/unittest.html#setupmodule-and-teardownmodule